skimage/restoration/_denoise.py

```
@@ -64,6 +68,20 @@ def denoise_bilateral(image, win_size=5, sigma_range=None, sigma_spatial=1,
     >>> noisy = np.clip(noisy, 0, 1)
     >>> denoised = denoise_bilateral(noisy, sigma_range=0.05, sigma_spatial=15)
     """
+    if multichannel:
+        if image.shape[2] not in (3, 4):
+            msg = "denoise_bilateral is expected to work correctly for " \
+                  "2D grayscale or RGB image; image shape {0} looks like " \
+                  "a 3D image."
+            warnings.warn(msg.format(image.shape))
+    else:
+        if image.ndim > 2:
+            msg = "denoise_bilateral is expected to work correctly for " \
```

Here we should raise an error, since we cannot handle ``multichannel=True``.